protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -4705,6 +4705,9 @@ message EstimateTransferFeeRequest {
   string amount = 4;
   // Asset ID for the asset associated with the transaction.
   string asset_id = 5;
+  // Optional saction must be the owner of this AMM key key from which assets are transferred.
```

Comment:
What is this line meant to say?

Reply:
@karlem We might be looking at different lines - I'm looking at " Optional saction must be the owner of this AMM key key from which assets are transferred."

Reply:
Yes. Could you please help me fix it. it sounds like rubbish sorry haha

Reply:
@karlem Ok - I made a suggestion based on what I think it's doing here